.circleci/config.yml

```diff
@@ -28,6 +28,8 @@ jobs:
 
             corepack enable
             corepack prepare yarn@stable --activate
+            npm install -g coveralls-lcov
```

Comment:
@harshkpatel you probably want https://github.com/nickmerwin/node-coveralls

It seems `coveralls-lcov` is a Ruby package